spotbugs/src/main/java/edu/umd/cs/findbugs/Version.java

```diff
@@ -99,22 +105,22 @@ public static void main(String[] argv) throws InterruptedException {
         String arg = argv[0];
 
         if ("-release".equals(arg)) {
-            System.out.println(VERSION_STRING);
+            LOGGER.info(VERSION_STRING);
         } else if ("-plugins".equals(arg)) {
             DetectorFactoryCollection.instance();
             for(Plugin p : Plugin.getAllPlugins()) {
-                System.out.println("Plugin: " + p.getPluginId());
-                System.out.println("  description: " + p.getShortDescription());
-                System.out.println("     provider: " + p.getProvider());
+                LOGGER.log(Level.INFO, "Plugin: {0}", p.getPluginId());
```

Comment:
I would use  `static import Level.*`, to be able to write it like:
`LOG.log(INFO, "Plugin: {0}", p.getPluginId());`

Reply:
I guess it's a matter of style. Is there any risk of `Level` constants clashing with anything else we might want to static-import (I'm particularly thinking of ERROR)?